src/nvim/lua/executor.c

```diff
@@ -474,7 +474,8 @@ static int nlua_wait(lua_State *lstate)
                             (int)timeout,
                             got_int || (is_function ? nlua_wait_condition(lstate,
                                                                           &pcall_status,
-                                                                          &callback_result) : false));
+                                                                          &callback_result) :
+                                        false));
```

Comment:
here's an example where ":" at the start is much more readable. one has to go to the end of the line to see where this `false` is coming from.
```suggestion
                                        : false));
```